server/events/project_command_runner.go

```diff
@@ -234,11 +234,7 @@ func (p *DefaultProjectCommandRunner) doApply(ctx models.ProjectCommandContext)
 				return "", "Pull request must be approved before running apply.", nil
 			}
 		case raw.MergeableApplyRequirement:
-			mergeable, err := p.PullMergeableChecker.PullIsMergeable(ctx.BaseRepo, ctx.Pull) // nolint: vetshadow
```

Comment:
The struct doesn't need the `PullMergeableChecker` anymore, and you can delete that whole `pull_mergeable_checker.go` file along with the `mocks/mock_pull_mergeable_checker.go` mock